app/views/content/blog/getting-ready-to-apply.md

```diff
@@ -52,7 +52,6 @@ Get one step closer to the classroom with guidance tailored to you, including ho
 An adviser with years of teaching experience can help you if you're (one of the following):
```

Comment:
This doesn't really make sense now @philspray - there is only one thing in the list!

Need to remove `(one of the following) and merge the bullet into the sentence?

(I know this blog will be deleted soon anyway)